HISTORY.rst

```diff
@@ -2,6 +2,46 @@
 History
 =======
 
+0.7.0 (2024-09-20)
+------------------
+* Features
+
+  * Signals and TimeData objects can now have complex time data. This is specified by the `complex` flag. In the case of Signal objects, this results in both sided spectra (PR #504, #396, #462, #464, #505, #510, #513, #547, #646, #655, #668, #704)
+    * Functions in the dsp module can now handle complex valued time data or raise an error (PR #459, #460, #461, #525, #546, #566, #567, #573, #592, #594, #598, #601, #677)
+    * All plot functions can now handle complex valued time data and both sided spectra via the options `side` and `mode`  (PR #615, #652, #656, #671, #672, #702)
+  * Added TransmissionMatrix Class (PR #636)
+  * Added frequency domain sweep synthesis (PR #199)
+  * Added spectral soft limiting(PR #325)
+  * Add arithmetic operations for Coordinate class (PR #680)
+  * Added transpose method to audio classes (PR #481)
+  * Added allpass filters (PR #571)
+  * Added `cdim`` property to audio classes (PR #619)
+
+* Enhancements
+
+  * The sampling_rate property of Signal Objects is enforced to be a scalar. (PR #534)
```

Comment:
isnt sth like sampling_rate=[1000] and sampling_rate=np.array([1000]) aslo allowed?

Reply:
I would say its allowed but not desired. And during initialization `signal = pf.Signal([1, 2, 3], np.array([1]))` `signal.sampling_rate` is caseted to `np.int`, which I think is the important enhancement.

Reply:
Yes, that makes sense. Thank you!